get-started/part3.md

```diff
@@ -146,10 +146,36 @@ Now let's run it. You have to give your app a name. Here, it is set to
 docker stack deploy -c docker-compose.yml getstartedlab
 ```
 
-See a list of the 5 containers you just launched:
+Our single service stack is running 5 container instances of our deployed image on one host. Let's investigate.
+
+Get the service ID for the one service in our application:
+
+```shell
+docker service ls
+```
+
+Swarm runs tasks that, in this case, spawn containers. Tasks have state and their own IDs:
```

Comment:
Why only "in this case"?

Reply:
Maybe this could be "in the case of Docker", with a clarification added somewhere (maybe a footnote) that tasks need not be containers in SwarmKit.